test/puppetlabs/puppetdb/cli/services_test.clj

```diff
@@ -572,15 +595,15 @@
        (fn []
          (testing "A report from 24 hours ago won't be gc'ed with a report-ttl of 1d"
            (store-report (-> 1 time/days time/ago time/to-string))
-           (svcs/sweep-reports! *db* {:incremental true
+           (svcs/sweep-reports! *db* {:incremental? false
```

Comment:
Also, also, this test has gc running at 0.01, which is probably good because it alerted me to the overlap between periodic and manual sweep, but I don't know if that was intentional?